### ISSUE TITLE
Possible issue with env variables

### ISSUE THREAD

**Issue by alexwhitmore** [withastro/astro#4730]
### What version of `astro` are you using?

1.2.2

### Are you using an SSR adapter? If so, which one?

None

### What package manager are you using?

npm

### What operating system are you using?

Mac

### Describe the Bug

> **Warning** 
> I am a noob at using environment variables and this is my first time trying in an Astro project or using Vite so it's possible that I may be doing something incorrectly.

I am following [this](https://docs.astro.build/en/guides/environment-variables/) Astro documentation for using .env files and variables. Essentially I have created a .env file in the root of my directory with the variable: `PUBLIC_ANYBODY=there` (I have also tried `PUBLIC_ANYBODY="there"`). In a separate file, in this case it's a .jsx file (using Solid), I am attempting to just console log the variable with the syntax of `console.log(import.meta.env.PUBLIC_ANYBODY);` and the result is that on both the sever and client side I am getting `undefined`.

I should add that I also attempted to log this in a .astro file and still was only able to return `undefined`.

I reached out to the support Discord thread and was pointed in the direction of raising an issue so I have attached a link to a reproduced example.

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/github-e4cvvq?file=src/.env

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by MykalMachon** [comment]
Hey 👋🏻 
I just gave this a quick look while checking out new issues. 

You just need to move your `.env` file up a folder! It should be in the root of your repo, not the root of your pages file. 
when I did that I was able to get things working 😄 

Check out [my fork of your StackBlitz repo](https://stackblitz.com/edit/github-e4cvvq-gfjhqd?file=src/Auth.jsx)

Let me know if you have any questions! (also sorry Astro peeps! don't know if this messed with your triage process)
